src/passes/rejectUnsupportedFeatures.ts

```diff
@@ -113,10 +119,49 @@ export class RejectUnsupportedFeatures extends ASTMapper {
   }
 
   visitFunctionDefinition(node: FunctionDefinition, ast: AST): void {
+    if (isExternallyVisible(node)) {
+      node.vParameters.vParameters.forEach((decl) => {
+        const type = getNodeType(decl, ast.compilerVersion);
```

Comment:
just for the sake of maintaining the code structure , it's better to change the name  _type_  to  _typeNode_